eslint.config.mjs

```diff
@@ -403,216 +278,109 @@ export default typescript.config([
           block: {exceptions: ['*'], balanced: true},
         },
       ],
+      'vars-on-top': 'off',
+      'wrap-iife': ['error', 'any'],
+      radix: 'error',
+      strict: 'error',
+      yoda: 'error',
 
-      // Let formatter handle this
-      'arrow-body-style': 'off',
-
-      /**
-       * Restricted imports, e.g. deprecated libraries, etc
-       *
-       * See: https://eslint.org/docs/rules/no-restricted-imports
-       */
-      'no-restricted-imports': [
-        'error',
-        {
-          patterns: restrictedImportPatterns,
-          paths: restrictedImportPaths,
-        },
-      ],
-
-      // https://eslint.org/docs/rules/no-console
-      'no-console': 'error',
+      // https://github.com/eslint/eslint/blob/main/packages/js/src/configs/eslint-recommended.js
+      ...eslint.configs.recommended.rules,
```

Comment:
here i've enabled the recommended list

below is stuff in that is in that list, but is a problem in our codebase right now.